src/containers/App/App.js

```diff
@@ -15,28 +18,33 @@ class App extends Component {
       correctBrowser: true,
       isOwner: false,
       posts: [],
-      postDisplay: 'theirs'
+      theirPosts: [],
+      postDisplay: 'mine',
+      editProfile: false,
+      userData: {}
     };
   }
 
   async componentDidMount() {
     try {
       const archive = await new global.DatArchive(urlEnv());
       const archiveInfo = await archive.getInfo();
-      const allPosts = await this.getAllPosts(archive);
+      const results = await this.refreshPosts(archive);
```

Comment:
I think there's a faulty merge going on here, starting with this line. We want the getAllPosts stuff